index.js

```diff
@@ -111,11 +111,12 @@ class Collection extends BaseAPI {
 
   static getCollectionBySlug(client, slug, params, options = {}) {
     const {depth = DEFAULT_DEPTH} = options;
+    const storyFields = _.get(params, ["story-fields"], null);
```

Comment:
const storyFields = _.get(params, ["story-fields"]) || DEFAULT_STORY_FIELDS + ''

Reply:
Basically, call it additional story fields here. Else you will get confused why you pass one param here, and it changes to something else later.

Reply:
ok will update